src/Workspaces/Core/Portable/Workspace/Solution/ProjectState.cs

```diff
@@ -872,7 +872,7 @@ public ProjectState RemoveAllDocuments()
                 analyzerConfigSet: ComputeAnalyzerConfigOptionsValueSource(AnalyzerConfigDocumentStates));
         }
 
-        public ProjectState UpdateDocument(DocumentState newDocument, bool textChanged, bool recalculateDependentVersions)
+        public ProjectState UpdateDocument(DocumentState newDocument, bool contentChanged)
```

Comment:
merged these to one value.  if hte content changed we need to recalculateDependentVersions, and if it didn't, we don't.